pkg/router/router.go

```diff
@@ -31,8 +30,7 @@ func (dr *DumbRouter) SetHeaders(w http.ResponseWriter) {
 // RootHandler is a simple http hander for / that returns "Go-Camo"
 func (dr *DumbRouter) RootHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
-	// Status 200 is the default. No need to set explicitly here.
-	io.WriteString(w, dr.ServerName)
+	http.Error(w, "404 Not found", 404)
```

Comment:
the main benefit of returning a 200 here vs a 404, is having a target that a load balancer can use to ensure the backend is up and running. I'll think about this one a bit more deeply before giving a yay or nay.

Reply:
With our internal fork, we exposed a `/status` route for the load balancer to handle this scenario. I'm happy to bring that over too if you see merit in that?

Reply:
/status is optional (and disabled by default), so I don't think that would work well.

Reply:
Ah. I think I see the confusion here. There is an internal stats handler (optional) that is also `/status`. 
It arguably should have been named `/stats`, but I'm not sure it is worth a breaking change to modify it.

Reply:
Any load balancer place in front of go-camo should be able to respond with a 404 to the root route as well, while still using the root route as a health-check on the backend.
I would be fine changing this to an http 204 empty response though (instead of returning the server name).

Reply:
> It arguably should have been named `/stats`, but I'm not sure it is worth a breaking change to modify it.

What about `/healthcheck`? I'd really like to separate the root route and the backend status check responsibilities and make the routes explicit.

> Any load balancer place in front of go-camo should be able to respond with a 404 to the root route as well, while still using the root route as a health-check on the backend.

I'm attempting to limit leaking this type of routing knowledge into the load balancer and leave it in the application for a couple of reasons:

- Not everyone has access to perform routing at the load balancer (or perhaps even a load balancer);
- It is muddying the waters of which part of the system is responsible for what component by having it in two places.

Reply:
> What about /healthcheck? 

Sure. That works.

Reply:
Added `/healthcheck` in 171c10c and removed the root handler